media-sound/alacenc/alacenc-0.4.0.ebuild

```diff
@@ -15,6 +15,16 @@ KEYWORDS="~amd64 ~x86"
 
 BDEPEND="virtual/pkgconfig"
 
+src_prepare() {
+	default
+
+	if use elibc_musl; then
```

Comment:
Not a fan of conditional patches, please make this patch global.

Reply:
Thanks, that makes sense. The patch works with musl as well as glibc.